backend/routes/events.py

```
@@ -0,0 +1,63 @@
+from typing import Dict, List
+from schemas.post import  EventResponse
+from models.user import User
+from database.session import SessionLocal
+from core.dependencies import get_db
+from api.v1.endpoints.auth import get_current_user
+
+from models.post import Event
+
+router = APIRouter()
+
+@router.get("/events/grouped-by-time", response_model=Dict[str, List[int]])
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)